### PR TITLE
fix($marks): Fix issue with width of mark

### PR DESCRIPTION
**Pull request by jeznag**
Mark width was too wide (90%) so if you clicked outside the mark, it moved the handle

#286

### CODE REVIEW

**Comment by benjycui** [comment]
Could you post a screenshot here?

File: assets/index.less [hunk 1]
```diff
@@ -80,6 +80,7 @@
   }
 
   &-mark-text {
+    margin-left: -10px;
```
Comment (paranoidjk) [inline assets/index.less]: text length is variable

**Comment by jeznag** [comment]
image
Screenshot of fixed bug. I'm not sure if there's a better way to calculate the width of the text.

File: README.md [hunk 2]
```diff
@@ -97,6 +97,7 @@ The following APIs are shared by Slider and Range.
 | min | number | `0` | The minimum value of the slider |
 | max | number | `100` | The maximum value of the slider |
 | marks | `{number: string}` or`{number: { style, label }}` | `{}` | Marks on the slider. The key determines the position, and the value determines what will show. If you want to set the style of a specific mark point, the value should be an object which contains `style` and `label` properties. |
+| markFontSize | number | 10 | The font size used for the mark labels. This is used to calculate the width of the mark labels. |
```
Comment (benjycui) [inline README.md]: I don't think this is a good idea, the actual font size is out of control.

Comment (jeznag) [reply README.md]: I don't know how else we can accurately calculate the width of a mark. Any suggestions?

**Comment by benjycui** [comment]
👎 for this, actually, we can solve this issue, but it is expensive(need to re-calculate marks' width every time). And this issue is not so troublesome.

**Pull request closed without merging**